### DISCUSSION TITLE
Error: macro01.tcl, 10 couldn't open "macro01.defok": no such file or directory

### DISCUSSION THREAD

**Discussion by ReemAbuRajab7** [The-OpenROAD-Project/OpenROAD#6091] [Q&A]
I solve it by change 'macro01.tcl ' from ($test_name.defok) to ($test_name.def), and it worked , is that a right solution ? 
![Screenshot from 2024-11-04 23-54-10](https://github.com/user-attachments/assets/e26f5151-2a98-4aa1-94da-63cfd678a2f5)
![Screenshot from 2024-11-04 23-53-16](https://github.com/user-attachments/assets/0028aa5c-3ab3-425c-a416-7f5b6d059c18)
![Screenshot from 2024-11-04 23-46-19](https://github.com/user-attachments/assets/bff4e627-d34f-4e54-b655-64994e72a0b0)

**Comment by maliberty** [comment]
This test appears not to be part of the regression.  Why are you running it?

**Reply by ReemAbuRajab7** [reply]
I don't understand what do you mean 😅 , I just followed the flow-tutorial in the documentation and found this error .

**Reply by maliberty** [reply]
Where is the flow tutorial does it mention running these tests?

**Reply by ReemAbuRajab7** [reply]
https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html#understanding-and-analyzing-openroad-flow-stages-and-results
![Screenshot_2024-11-05-16-09-39-549_com android chrome](https://github.com/user-attachments/assets/13f33734-8079-4fbf-9550-02a4ef86b458)

**Comment by maliberty** [comment]
I don't think it is the best example but since its in the docs I've made them run in https://github.com/The-OpenROAD-Project/OpenROAD/pull/6100
